docs/vault/upgrading.md

```diff
@@ -120,3 +120,100 @@ sudo systemctl start kintsugi-vault.service
 <!-- tabs:end -->
 
 OR start the [process manually](vault/installation?id=_5-start-the-vault-client).
+
+## Vault Auto-Updater
+
+Vaults can be run with [Runner](https://github.com/interlay/interbtc-clients/tree/master/runner) to automate updates. It reads on-chan release data (the `ClientsInfo::CurrentClientsRelease` storage map), downloads and runs new binaries as soon as chain storage is updated. The Runner restarts the vault on crash, thus also fulfilling the function of a Vault systemd service.
+
+### 1. Install from source
```

Comment:
Would it be possible to build a binary for this and we refer to GitHub for manual install and docs would just use the binary?